qt-models/diveplannermodel.cpp

```diff
@@ -360,8 +360,10 @@ void DivePlannerPointsModel::gasChange(const QModelIndex &index, int newcylinder
 
 void DivePlannerPointsModel::cylinderRenumber(int mapping[])
 {
-	for (int i = 0; i < rowCount(); i++)
-		divepoints[i].cylinderid = mapping[divepoints[i].cylinderid];
+	for (int i = 0; i < rowCount(); i++) {
+		if (mapping[divepoints[i].cylinderid] >= 0)
+			divepoints[i].cylinderid = mapping[divepoints[i].cylinderid];
+		}
```

Comment:
would this `}` close the `for` loop?

Reply:
Yep, indentation wrong. Will fix this... Other findings?

Reply:
nope, can't seem to find anything else.